talkgenerator/server/hello.py

```diff
@@ -10,33 +10,37 @@
 sslify = SSLify(app)
 
 app.config.from_object(__name__)
-app.config['SECRET_KEY'] = '101710171017'
-app.config['OUTPUT_FOLDER'] = 'output'
+app.config["SECRET_KEY"] = "101710171017"
+app.config["OUTPUT_FOLDER"] = "output"
+
 
 class ReusableForm(Form):
-    talk_topic = TextField('Topic: ', validators=[validators.required()])
+    talk_topic = TextField("Topic: ", validators=[validators.required()])
+
 
-@app.route('/output/<path:filename>', methods=['GET', 'POST'])
+@app.route("/output/<path:filename>", methods=["GET", "POST"])
 def download(filename):
-    outputs = os.path.join(app.root_path, app.config['OUTPUT_FOLDER'])
+    outputs = os.path.join(app.root_path, app.config["OUTPUT_FOLDER"])
     return send_from_directory(directory=outputs, filename=filename)
 
-@app.route("/", methods=['GET', 'POST'])
+
+@app.route("/", methods=["GET", "POST"])
 def hello():
     form = ReusableForm(request.form)
-    print('Errors: {}'.format(form.errors))
+    print("Errors: {}".format(form.errors))
```

Comment:
Change to logging?

Reply:
All of those are of files we want to remove, so it felt like quite unnecessary work :)

Reply:
Agree. LGTM.